### PR TITLE
Flight calendar2

### LINKED ISSUE #99: UI: Flight calendar screen
Create and test the screen of the calendar of the flights

### PR DESCRIPTION
**Pull request by Zep3k**
Screenshot_20240411-192741_SkySync
Close #99

### CODE REVIEW

**Review by Baribal02 — APPROVED** [review]
The AddFlight.kt and AddFlightTest.kt will be replaced by the version in https://github.com/SkySync-EPFL/skysync/pull/119. We should make sure during the Route is the same and change it if needed.
Overall well done !

File: app/src/main/java/ch/epfl/skysync/screens/FlightCalendar.kt [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  ) {
+    WeekView(currentWeekStartDate, viewModel, navController)
+    Spacer(modifier = Modifier.height(8.dp))
+    Row(
+        modifier = Modifier.fillMaxWidth().padding(vertical = 10.dp),
+        horizontalArrangement = Arrangement.SpaceBetween) {
+          Spacer(modifier = Modifier.width(5.dp))
+          Button(
+              onClick = { currentWeekStartDate = currentWeekStartDate.minusWeeks(1) },
+              colors = ButtonDefaults.buttonColors(containerColor = Color(0xFFFFA500))) {
```
Comment (saraanej) [inline app/src/main/java/ch/epfl/skysync/screens/FlightCalendar.kt:115]: For raw values like code colors, it is strongly advised to use global variables that you initialize at the beginning of the code for better modularity :)

File: app/src/androidTest/java/ch/epfl/skysync/FlightCalendarTest.kt
Comment (saraanej) [inline app/src/androidTest/java/ch/epfl/skysync/FlightCalendarTest.kt:1]: You should maybe add a test that makes sure the initial UI is visible before doing any action. This helps improve the test coverage :)

**Review by saraanej — COMMENTED** [review]
The code is very clean and well documented ! I had some minor comments for improvement.   Good job for the work.

**Pull request merged**